### PR TITLE
Issue17 post

### PR DESCRIPTION
**Pull request by technetos**

### CODE REVIEW

File: calendar/models/user.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+var db = require('../config/db');
```
Comment (DannyPeck) [inline calendar/models/user.js]: I would move the model line into the schema file and export the model.

File: calendar/app.js [hunk 2]
```diff
@@ -1,19 +1,12 @@
 let express = require('express');
-let path = require('path');
-let favicon = require('serve-favicon');
-let logger = require('morgan');
```
Comment (nrhubbar) [inline calendar/app.js]: Why did we get rid of morgan?

File: calendar/app.js [hunk 3]
```diff
@@ -1,19 +1,12 @@
...
-let cookieParser = require('cookie-parser');
 let bodyParser = require('body-parser');
 
+let userController = require('./controllers/user-controller');
```
Comment (nrhubbar) [inline calendar/app.js]: Can we move this require closer to where we use it? and should it be const?

Comment (technetos) [inline calendar/app.js]: no these need to be before let app = require('express')

File: calendar/app.js (hunk 2 above)
Comment (technetos) [inline calendar/app.js]: we are not using them at all

File: calendar/app.js (hunk 3 above)
Comment (technetos) [inline calendar/app.js]: lol or not

**Pull request merged**